libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -163,6 +164,15 @@ const AP_Param::GroupInfo AP_Arming::var_info[] = {
     AP_GROUPINFO("CRSDP_IGN", 11, AP_Arming, crashdump_ack.acked, 0),
 #endif  // AP_ARMING_CRASHDUMP_ACK_ENABLED
 
+#if HAL_WITH_ESC_TELEM
+    // @Param: ESC_CHECK
+    // @DisplayName: Required ESC Telemetry Check
+    // @Description: Check if ESC telemetry is required to arm the vehicle
+    // @Bitmask: 0:ESC 1,1:ESC 2,2:ESC 3,3:ESC 4,4:ESC 5,5:ESC 6,6:ESC 7,7:ESC 8,8:ESC 9,9:ESC 10,10:ESC 11,11:ESC 12,12:ESC 13,13:ESC 14,14:ESC 15,15:ESC 16,16:ESC 17
```

Comment:
18 ESC seems pretty arbitrary. I think you should probably do 16 and then have another param for the other 15 if 32 servos are enabled.

Also I'm not sure this is necessary - for TKOFF_MIN_RPM we assume that configured motors must have ESCs - if it really is necessary then needs to go in as an ESC telemetry option rather than arming option